docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,30 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test alternative deployments and options.
```

Comment:
Haha ok will do.  I'll just put  this data in a message to the mailing list or an issue or something....